infra/relay.ts

```diff
@@ -0,0 +1,73 @@
+import { resolve as pathResolve } from "node:path";
+import { readFileSync as readFile } from "node:fs";
+//Copy your (known) ssh public key to the remote machine
+//ssh-copy-id "-p $port" user@host
+
+const ipv4Address = "95.216.29.238"
+const user = "wanjodotryan"
+const port = 789
+const privateKey = readFile(pathResolve("./id_ed25519"))
+
+// Connect to the Docker Server on the Hetzner Server
+const dockerServer = new docker.Provider("Docker Server - Hetzner", {
```

Comment:
This is getting too crazy hard to maintain, and it's goes against the Nestri's ethos of keeping everything easy and composable.

Consider other options

Reply:
Also Removed